### PR TITLE
fix(scripts): v9 release should not include v8 docsite in scope

### PR DESCRIPTION
**Pull request by ling1726**
The current scoping means that v8 docsites will always be in scope for v9 release

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                          |         Size (minified/GZIP) |
| -------------------------------------------------------------------------------------------------------------------------- | ---------------------------: |
| <samp>global-context</samp> <br /> <abbr title='bundle-size/createContext.fixture.js'>createContext</abbr>                 |         `533 B`<br />`341 B` |
| <samp>global-context</samp> <br /> <abbr title='bundle-size/createContextSelector.fixture.js'>createContextSelector</abbr> |         `554 B`<br />`348 B` |
| <samp>priority-overflow</samp> <br /> <abbr title='bundle-size/overflow-manager.fixture.js'>createOverflowManager</abbr>   |   `2.936 kB`<br />`1.212 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/Badge.fixture.js'>Badge</abbr>                                    |    `22.6 kB`<br />`7.205 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/CounterBadge.fixture.js'>CounterBadge</abbr>                      |  `23.503 kB`<br />`7.497 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/PresenseBadge.fixture.js'>PresenceBadge</abbr>                    |   `24.05 kB`<br />`7.067 kB` |
| <samp>react-divider</samp> <br /> <abbr title='bundle-size/Divider.fixture.js'>Divider</abbr>                              |  `16.459 kB`<br />`5.902 kB` |
| <samp>react-image</samp> <br /> <abbr title='bundle-size/Image.fixture.js'>Image</abbr>                                    |   `10.78 kB`<br />`4.264 kB` |
| <samp>react-input</samp> <br /> <abbr title='bundle-size/Input.fixture.js'>Input</abbr>                                    |  `23.757 kB`<br />`7.704 kB` |
| <samp>react-label</samp> <br /> <abbr title='bundle-size/Label.fixture.js'>Label</abbr>                                    |    `9.338 kB`<br />`3.86 kB` |
| <samp>react-link</samp> <br /> <abbr title='bundle-size/Label.fixture.js'>Link</abbr>                                      |  `11.784 kB`<br />`4.867 kB` |
| <samp>react-overflow</samp> <br /> <abbr title='bundle-size/hooks.fixture.js'>hooks only</abbr>                            |  `10.685 kB`<br />`4.104 kB` |
| <samp>react-portal</samp> <br /> <abbr title='bundle-size/Portal.fixture.js'>Portal</abbr>                                 |  `10.576 kB`<br />`3.875 kB` |
| <samp>react-positioning</samp> <br /> <abbr title='bundle-size/usePopper.fixture.js'>usePositioning</abbr>                 |    `19.7 kB`<br />`7.404 kB` |
| <samp>react-provider</samp> <br /> <abbr title='bundle-size/FluentProvider.fixture.js'>FluentProvider</abbr>               |  `15.755 kB`<br />`5.883 kB` |
| <samp>react-select</samp> <br /> <abbr title='bundle-size/Select.fixture.js'>Select</abbr>                                 |  `20.846 kB`<br />`7.346 kB` |
| <samp>react-slider</samp> <br /> <abbr title='bundle-size/Slider.fixture.js'>Slider</abbr>                                 | `31.526 kB`<br />`10.046 kB` |
| <samp>react-spinner</samp> <br /> <abbr title='bundle-size/Spinner.fixture.js'>Spinner</abbr>                              |  `19.977 kB`<br />`6.438 kB` |
| <samp>react-switch</samp> <br /> <abbr title='bundle-size/Switch.fixture.js'>Switch</abbr>                                 |  `32.097 kB`<br />`10.27 kB` |
| <samp>react-text</samp> <br /> <abbr title='bundle-size/Text.Default.fixture.js'>Text - Default</abbr>                     |  `11.782 kB`<br />`4.605 kB` |
| <samp>react-text</samp> <br /> <abbr title='bundle-size/Text.Wrappers.fixture.js'>Text - Wrappers</abbr>                   |  `15.092 kB`<br />`5.044 kB` |
| <samp>react-textarea</samp> <br /> <abbr title='bundle-size/Textarea.fixture.js'>Textarea</abbr>                           |  `23.988 kB`<br />`8.011 kB` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/SingleThemeToken.fixture.js'>Single theme token import</abbr>     |           `69 B`<br />`89 B` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/TeamsAllThemes.fixture.js'>Teams: all themes</abbr>               |   `29.65 kB`<br />`6.444 kB` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/TeamsLightTheme.fixture.js'>Teams: Light theme</abbr>             |  `17.486 kB`<br />`5.057 kB` |
| <samp>react-utilities</samp> <br /> <abbr title='bundle-size/SSRProvider.fixture.js'>SSRProvider</abbr>                    |         `180 B`<br />`159 B` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/f5398157ebbea5c748d8d7336f18b378a8bdec16'>f5398157ebbea5c748d8d7336f18b378a8bdec16</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test-react-components/Avatar-mount_base.html">1281</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test-react-components/Avatar-mount.html">1276</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test-react-components/Button-mount_base.html">931</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test-react-components/Button-mount.html">943</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test-react-components/FluentProvider-mount_base.html">1487</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test-react-components/FluentProvider-mount.html">1504</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test-react-components/FluentProviderWithTheme-mount_base.html">576</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test-react-components/FluentProviderWithTheme-mount.html">578</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">546</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">547</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">572</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">580</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test-react-components/MakeStyles-mount_base.html">1948</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test-react-components/MakeStyles-mount.html">1947</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test-react-components/SpinButton-mount_base.html">2359</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test-react-components/SpinButton-mount.html">2332</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

File: scripts/monorepo/runPublished.js [hunk 1]
```diff
@@ -23,23 +23,26 @@ const websitePackages = [
...
     const isNorthstar = /[\\/]fluentui[\\/]/.test(packagePath);
 
+    // Northstar has separate release tooling, always ignore northstar
     if (isNorthstar) {
       return false;
     }
 
     const isConverged = isConvergedPackage({ packagePathOrJson: packageJson });
-    if (process.env.RELEASE_VNEXT && isConverged) {
+    if (releaseScope === 'v9' && isConverged) {
```
Comment (Hotell) [inline scripts/monorepo/runPublished.js:41]: can we unify this one way or another for both if statements?
`releaseScope === 'v9' && isConverged` or `isConverged ** releaseScope === 'v9'`

File: scripts/monorepo/runPublished.js [hunk 2]
```diff
@@ -23,23 +23,26 @@ const websitePackages = [
   '@fluentui/api-docs',
 ];
 
+const releaseScope = process.env.RELEASE_VNEXT ? 'v9' : 'v8';
```
Comment (Hotell) [inline scripts/monorepo/runPublished.js:26]: can we add more docs to this  ? there is something for beachbalPackagesScopes but Im not sure its up to date

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/BaseButton-mount_base.html">768</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/BaseButton-mount.html">794</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/Breadcrumb-mount_base.html">2320</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/Breadcrumb-mount.html">2330</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/Checkbox-mount_base.html">2246</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/Checkbox-mount.html">2206</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/CheckboxBase-mount_base.html">1949</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/CheckboxBase-mount.html">1930</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/ChoiceGroup-mount_base.html">3815</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/ChoiceGroup-mount.html">3804</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/ComboBox-mount_base.html">758</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/ComboBox-mount.html">763</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/CommandBar-mount_base.html">8779</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/CommandBar-mount.html">8816</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/ContextualMenu-mount_base.html">9622</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/ContextualMenu-mount.html">9598</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/DefaultButton-mount_base.html">939</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/DefaultButton-mount.html">943</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/DetailsRow-mount_base.html">2991</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/DetailsRow-mount.html">2898</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/DetailsRowFast-mount_base.html">2952</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/DetailsRowFast-mount.html">2975</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/DetailsRowNoStyles-mount_base.html">2811</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/DetailsRowNoStyles-mount.html">2799</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/Dialog-mount_base.html">2487</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/Dialog-mount.html">2510</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/DocumentCardTitle-mount_base.html">150</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/DocumentCardTitle-mount.html">163</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/Dropdown-mount_base.html">2734</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/Dropdown-mount.html">2774</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/FocusTrapZone-mount_base.html">1563</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/FocusTrapZone-mount.html">1550</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/FocusZone-mount_base.html">1520</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/FocusZone-mount.html">1512</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/IconButton-mount_base.html">1359</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/IconButton-mount.html">1356</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/Label-mount_base.html">322</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/Label-mount.html">319</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/Layer-mount_base.html">3702</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/Layer-mount.html">3694</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/Link-mount_base.html">431</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/Link-mount.html">430</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/MenuButton-mount_base.html">1199</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/MenuButton-mount.html">1209</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/MessageBar-mount_base.html">1884</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/MessageBar-mount.html">1886</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/Nav-mount_base.html">2639</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/Nav-mount.html">2657</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/OverflowSet-mount_base.html">967</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/OverflowSet-mount.html">999</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/Panel-mount_base.html">2088</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/Panel-mount.html">2069</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/Persona-mount_base.html">854</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/Persona-mount.html">827</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/Pivot-mount_base.html">1111</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/Pivot-mount.html">1118</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/PrimaryButton-mount_base.html">1062</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/PrimaryButton-mount.html">1066</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/Rating-mount_base.html">6611</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/Rating-mount.html">6546</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/SearchBox-mount_base.html">1070</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/SearchBox-mount.html">1085</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/Shimmer-mount_base.html">2487</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/Shimmer-mount.html">2488</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/Slider-mount_base.html">1675</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/Slider-mount.html">1689</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/SpinButton-mount_base.html">3797</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/SpinButton-mount.html">3889</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/Spinner-mount_base.html">400</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/Spinner-mount.html">400</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/SplitButton-mount_base.html">2424</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/SplitButton-mount.html">2431</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/Stack-mount_base.html">456</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/Stack-mount.html">465</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/StackWithIntrinsicChildren-mount_base.html">1842</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/StackWithIntrinsicChildren-mount.html">1827</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/StackWithTextChildren-mount_base.html">4494</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/StackWithTextChildren-mount.html">4507</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/SwatchColorPicker-mount_base.html">9061</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/SwatchColorPicker-mount.html">9057</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/TagPicker-mount_base.html">1929</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/TagPicker-mount.html">1946</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/TeachingBubble-mount_base.html">73987</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/TeachingBubble-mount.html">73657</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/Text-mount_base.html">396</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/Text-mount.html">397</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/TextField-mount_base.html">1129</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/TextField-mount.html">1133</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/ThemeProvider-mount_base.html">1022</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/ThemeProvider-mount.html">1016</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/ThemeProvider-virtual-rerender_base.html">711</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/ThemeProvider-virtual-rerender.html">724</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">1577</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">1580</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/Toggle-mount_base.html">710</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/Toggle-mount.html">717</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/buttonNative-mount_base.html">115</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/24809/perf-test/buttonNative-mount.html">101</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

**Review by Hotell — APPROVED** [review]

File: scripts/monorepo/runPublished.js [hunk 3]
```diff
@@ -23,23 +23,26 @@ const websitePackages = [
...
     }
 
     const isConverged = isConvergedPackage({ packagePathOrJson: packageJson });
-    if (process.env.RELEASE_VNEXT && isConverged) {
+    if (releaseScope === 'v9' && isConverged) {
       return packageJson.private !== true;
     }
 
-    if (!isConverged) {
+    if (!isConverged && releaseScope === 'v8') {
```
Comment (Hotell) [inline scripts/monorepo/runPublished.js:45]:
```suggestion
    if (releaseScope === 'v8' && !isConverged) {
```

**Comment by Hotell** [comment]
I added this fix in this bigger PR https://github.com/microsoft/fluentui/pull/25095

**Pull request closed without merging**